### PR TITLE
Add repo owners, status descriptions and milestones

### PR DESCRIPTION
**Pull request by dseefeld**

### CODE REVIEW

File: Documentation/planning/arcade-powered-source-build/implementation-plan.md [hunk 1]
```diff
@@ -40,3 +46,14 @@ To get each repo building with the new source-build 5.0 plan, [Arcade-Powered So
   - **(5) Official build** – Build source-built intermediate packages in official build.
   - **(6) Source-build PR validation** – Add AzDO build jobs to PR validation to ensure build-from-source continues running without avoidable regressions.
   - **(7) Prebuilt check / enforcement** – Add a check to PR validation to ensure that no prebuilts are included when building repo.
+
+## Key Milestone Dates:
+
+| Stage(s) | Start | End |
+| --- | :---: | :---: |
+| Stage 1 - Build from Source 5.0 | 5/28 | 6/29 |
```
Comment (dagood) [inline Documentation/planning/arcade-powered-source-build/implementation-plan.md:54]: YYYY/MM/DD for not-necessarily-US?

File: Documentation/planning/arcade-powered-source-build/implementation-plan.md [hunk 2]
```diff
@@ -5,32 +5,38 @@ To get each repo building with the new source-build 5.0 plan, [Arcade-Powered So
...
+| Tier | Repo | Owner | (Stage 1)<br>Build from source - 5.0 | (Stage 2)<br>Patch Removal | (Stage 3)<br>Move build params to eng dir | (Stage 4)<br>Input Intermediate Packages Available | (Stage 5)<br>Official build | (Stage 6)<br>Source-build PR validation | (Stage 7)<br>Prebuilt check / enforcement |
+| --- | --- | --- | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
+| sbrp | Source-build-reference-packages | Dan Seefeldt | ✔️ | ✔️ | | | | | |
+| Tools | sourcelink | Tomas Matousek | ✔️ | 4 | | | | | |
+| Tools | arcade | Mark Wilkie | ✔️ | 11 | | | | | |
+| 1 | application-insights | Reiley Yang | ✔️ | ✔️ | | | | | |
+| 1 | aspnet-xdt | Vijay Ramakrishnan | ✔️ | 2 | | | | | |
+| 1 | newtonsoft-json | Chris Rummel | ✔️ | ✔️ | | | | | |
+| 1 | netcorecli-fsc | Chris Rummel | ✔️ | ✔️ | | | | | |
+| 1 | newtonsoft-json901 | Chris Rummel | ✔️ | ✔️ | | | | | |
```
Comment (dagood) [inline Documentation/planning/arcade-powered-source-build/implementation-plan.md:17]: Add GitHub alias, or make the name a link to the GitHub account to save space?

Comment (dseefeld) [reply Documentation/planning/arcade-powered-source-build/implementation-plan.md:17]: I thought about using aliases, but thought I liked full names better.

Comment (dagood) [reply Documentation/planning/arcade-powered-source-build/implementation-plan.md:17]: Did you consider linking to the account so the alias is at least accessible?

Comment (dseefeld) [reply Documentation/planning/arcade-powered-source-build/implementation-plan.md:17]: https://github.com/dotnet/source-build/pull/1623

**Pull request merged**